packages/code-studio/src/settings/SettingsMenu.tsx

```diff
@@ -345,6 +346,10 @@ export class SettingsMenu extends Component<
                   </Tooltip>
                 </span>
               </div>
+              <div className="app-settings-footer-item">
+                <div className="font-weight-bold">Plugins</div>
+                <PluginInformation />
+              </div>
```

Comment:
It's kinda useful to expose I guess. But probably should be in both places.

How does it handle extremely long plugin names? Does it truncate nicely?

Reply:
@dsmmcken 

> It's kinda useful to expose I guess. But probably should be in both places.

So it should have its own section as shown in the screenshot, and then should be also in the "Versions" tooltip?

> 
> How does it handle extremely long plugin names? Does it truncate nicely?

I can add the truncation css styling, where it will truncate with the ellipsis, would you have any other preference?